roles/test_operator/tasks/run-test-operator-job.yml

```diff
@@ -110,8 +110,8 @@
         wait: true
```

Comment:
Can we not instead use a `wait_condition` in the k8s_info task? Like we do in other places, for example: https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/kustomize_deploy/tasks/install_operators.yml#L118-L129

Reply:
+1 To this thing, that's the proper way to do so and how we usually wait for a pod.